front/src/__tests__/hooks/useGetCategories.test.tsx

```diff
@@ -0,0 +1,47 @@
+import { QueryClientProvider } from '@tanstack/react-query'
+import { vi } from 'vitest'
+import { useGetCategories } from '../../hooks/useGetCategories'
+import { renderHook, waitFor } from '../test-utils'
+import { queryClient } from '../setup'
+
+describe('useGetCategories', () => {
+  beforeEach(() => {
```

Comment:
Probar con msw, ejemplo: https://github.com/IT-Academy-BCN/ita-wiki/blob/main/front/src/__tests__/hooks/useGetTopics.test.tsx